.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -10,6 +10,7 @@ IS_TEST_EXECUTION_STEP="$(buildkite-agent meta-data get "${BUILDKITE_JOB_ID}_is_
 
 if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
   echo "--- Upload Artifacts"
+  buildkite-agent artifact upload '.scout/performance/**/*'
```

Comment:
We talked offline, and this line will be removed because we will be uploading artifacts to Buildkite to a separate PR.